doc/release/1.14.0-notes.rst

```diff
@@ -253,37 +253,50 @@ enabling the new 1.13 "legacy" printing mode. This is enabled by calling
 
 In summary, the major changes are:
 
-* The ``repr`` of float arrays often omits a whitespace previously printed
-  in the sign position. See the new ``sign`` option to ``np.set_printoptions``.
-* Floating-point arrays and scalars use a new algorithm for decimal
-  representations, giving the shortest unique representation. This will
-  usually shorten ``float16`` fractional output, and sometimes ``float32`` and
-  ``float128`` output. ``float64`` should be unaffected.  See the new
-  ``floatmode`` option to ``np.set_printoptions``.
-* Float arrays printed in scientific notation no longer use fixed-precision,
-  and now instead show the shortest unique representation.
-* The ``str`` of floating-point scalars is no longer truncated in python2.
-* Non-finite complex scalars print like ``nanj`` instead of ``nan*j``.
+* To floating-point types:
```

Comment:
Indenting the list results in a line down the left hand side when rendered for documentation , which is why it wasn't indented before. The sublists need to be separated by blank lines. If you view the file you can see some of these problems.

Reply:
Generally, "to" should be replaced by "for".

Reply:
Thanks for reminding me that github can render RST files.